task/eks-cluster-create/0.1/eks-cluster-create.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: eks-cluster-create
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "aws, eks"
+    tekton.dev/displayName: "EKS Cluster Create"
+spec:
+  description: |
+    Create an EKS cluster.
+
+    This Task can be used to create an EKS cluster in an AWS account and fetch a kubeconfig that
+    can be used (in a context with kubectl) to make requests to the cluster.
+  params:
+  - name: cluster-name
+  - name: version
+    default: "1.17"
+  - name: region
+    default: us-west-2
+  - name: zones
+    default: us-west-2a,us-west-2b,us-west-2c
+  - name: nodegroup-name
+    default: linux-nodes
+  - name: node-type
+    default: m5.xlarge
+  - name: desired-nodes
+    default: "4"
+  - name: min-nodes
+    default: "1"
+  - name: max-nodes
+    default: "4"
```

Comment:
can we add a short description for the params as well over here? Probably you can take reference from [here](https://github.com/tektoncd/catalog/blob/6e8b8197343d78c397d997d26aae44b19c1ea266/task/aws-cli/0.1/aws-cli.yaml#L21)

Reply:
fixed in cc49d35